core/src/tangram.cpp

```diff
@@ -44,6 +44,11 @@ std::unique_ptr<Labels> m_labels;
 std::unique_ptr<Skybox> m_skybox;
 std::unique_ptr<InputHandler> m_inputHandler;
 
+bool m_viewComplete = false;
+std::function<void()> m_viewCompleteCallback = [](){
```

Comment:
Yes, but I'm currently thinking about switching to the setViewCompletedCallback API that you suggeted